gammapy/spectrum/flux_point.py

```diff
@@ -84,6 +84,10 @@ class FluxPoints:
         flux_points = FluxPoints.read(filename)
         flux_points.plot()
 
+    Note: In order to reproduce the example you need the tests datasets folder.
```

Comment:
This example could be changed to `"$GAMMAPY_DATA/hawc_crab/HAWC19_flux_points.fits"`